### DISCUSSION TITLE
push notifications to mobile devices

### DISCUSSION THREAD

**Discussion by Big-Brummer** [andrei-tatar/node-red-contrib-smartnora#261] [Q&A] [answered]
I am looking for an example of how to receive notifications on my mobile device.
It would be great if someone here could write me a HOW TO.
Many thanks in advance

**Comment by andrei-tatar** [comment]
Did you follow the instructions from https://github.com/andrei-tatar/node-red-contrib-smartnora/blob/master/doc/nodes/notify/README.md ?

**Comment by Big-Brummer** [comment]
yes, of course but unfortunately I do not understand how or where the message is displayed.
I have implemented the example but when I run the inject node nothing happens neither in debug nor in NORA.
in my NORA accout I also don't see "Firebase [test group]".
What do I have to change to see something on Nora or my mobile ?

**Comment by Big-Brummer** [comment, marked as answer]
It works now.
I had to change "config" to an entry I also use for my buttons.you have created a great solution. my thanks and high compliments to you

**Comment by Fragmike** [comment]
Also looking for some examples how to inject individual messages to the notivy node and to override title and body. How does the Inject node have to look like? Any link to where i can read howto?
